demo/helm/rook-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: rook-operator
+version: 0.1.0
```

Comment:
The convention for Helm charts is different to the version numbering of the app/service in the chart. I found it a little confusing at first but it does allow for the development of the chart independently from the app/service version. That way a new value/resource/etc can be added to the chart without having to change the app version, rebuild container images, etc.